tests/fixtures/coref/experiment.json

```diff
@@ -15,8 +15,8 @@
     },
     "max_span_width": 5
   },
-  "train_data_path": "tests/fixtures/conll_2012/",
```

Comment:
Do we still need these old fixtures?  Are they used by the SRL model, or can they be removed?

Reply:
Unfortunately they are used by the SRL model, because that assumes only one document per file.